src/components/Tabs.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Tab } from './Tab';
+
+import './Tabs.scss';
+
+type Props = {
+  tabs: Tab[],
+  onTabSelected: (selectedTab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, onTabSelected }) => (
+  <div className="Tabs">
+    <nav className="Tabs__nav">
+      {tabs.map(tab => (
+        <div className="Tab" key={tab.id}>
+          <Tab
+            key={tab.id}
```

Comment:
I think this key is unnecessary

Reply:
No, will be an error in console, need unique key property